test/general.js

```diff
@@ -31,7 +31,7 @@ test(`autocomplete doesn't throw`, async ({ uno }) => {
 });
 
 test(`the HTML parser is sane`, async ({ uno }) => {
-  const { css } = await uno.generate(`<div class="before:content-['before-stuff'] sm:grid"></div>`);
+  const { css } = await uno.generate([`before:content-['before-stuff']`, 'sm:grid']);
```

Comment:
There was an issue with `"` and `'` in the same string, causing unocss to not generate any css for these classes.